testkit/src/main/java/org/apache/calcite/sql/parser/SqlParserTest.java

```diff
@@ -2006,7 +2006,7 @@ void checkPeriodPredicate(Checker checker) {
     expr("1-2+3*4/5/6-7")
         .ok("(((1 - 2) + (((3 * 4) / 5) / 6)) - 7)");
     expr("power(2,3)")
-        .ok("POWER(2, 3)");
+        .ok("`POWER`(2, 3)");
```

Comment:
these quotes look suspicious

Reply:
This will end up going through here:
https://github.com/apache/calcite/blob/main/core/src/main/java/org/apache/calcite/sql/SqlUtil.java#L393

and only look in `SqlStdOperatorTable` for the function.

Reply:
Then it's related to my previous question: should there be a variant of this function in the standard operator table?
If not, I guess this is fine.

Reply:
Is it possible to have a power function in the standard operator table and still have an override for PostgreSQL that is able to return a decimal in some cases?

Reply:
I don't know the answer to this question. But I know that Calcite can load multiple libraries, and I believe a function will be resolved by trying the libraries in order. I also am assuming that there is nothing special about the standard operator table. So if you load the postgres operator table first, perhaps the right POWER will be used?

A quick experiment should be able answer this question.

If this works, we should probably document that the right way to implement a dialect is to load the dialect specific operator table first.

Reply:
I found where the `ChainedSqlOperatorTable` is created and was able to give the library operator table precedence over the standard operator table.

With this change, no changes are needed to existing tests.